installers/1.6.2/chocolatey/spark-installer-windows/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,9 @@
+﻿$ErrorActionPreference = 'Stop'; # stop on all errors
+
+$packageName= 'spark-installer-windows' # arbitrary name for the package, used in messages
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = 'https://raw.githubusercontent.com/petabridge/spark-installer-windows/master/installers/1.6.2/install-spark1.6.2.ps1'
+$fileLocation = Join-Path $toolsDir 'install-spark1.6.2.ps1'
+
+Get-ChocolateyWebFile $packageName $fileLocation $url
```

Comment:
It's also better to split out this stuff into multiple packages, set proper dependencies, and not just what was done in the script.